src/rpm/builder.rs

```diff
@@ -1,4 +1,5 @@
 use std::collections::{BTreeMap, BTreeSet};
+use std::convert::TryInto;
```

Comment:
OOS: Should we consider moving to edition 2021? https://doc.rust-lang.org/core/prelude/rust_2021/index.html